### PR TITLE
FIX invoice form view

### PR DESCRIPTION
**Pull request by drl9319**
When I press invoice contract button the tree view is correct because show client tree but when enter in invoice the form view is the supplier form, with the new code the form view is for client.

### CODE REVIEW

File: contract_show_invoice/views/contract_view.xml [hunk 1]
```diff
@@ -6,10 +6,25 @@
             <field name="name">Invoices</field>
             <field name="res_model">account.invoice</field>
             <field name="domain">[('invoice_line.account_analytic_id','=',active_id)]</field>
-            <field name="view_id" ref="account.invoice_tree" />
+            <field name="view_id" eval="False"/>
+            <!-- <field name="view_id" ref="account.invoice_tree" /> -->
```
Comment (pedrobaeza) [inline contract_show_invoice/views/contract_view.xml]: Remove this line

File: contract_show_invoice/__openerp__.py [hunk 2]
```diff
@@ -22,7 +22,7 @@
 {
     'name': 'Contract Show Invoice',
     'summary': 'Button in contracts to show their invoices',
-    'version': '8.0.2.0.0',
+    'version': '8.0.3.0.0',
```
Comment (pedrobaeza) [inline contract_show_invoice/__openerp__.py]: 8.0.2.1.0 (https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#version-numbers)

**Comment by pedrobaeza** [comment]
Please sign CLA. https://odoo-community.org/page/cla

**Comment by drl9319** [comment]
@pedrobaeza  I was sent the email to info@odoo-community.org a few minutes ago. How much time delay to answer?

**Comment by rafaelbn** [comment]
:+1:

File: contract_show_invoice/views/contract_view.xml [hunk 3]
```diff
@@ -6,10 +6,24 @@
             <field name="name">Invoices</field>
             <field name="res_model">account.invoice</field>
             <field name="domain">[('invoice_line.account_analytic_id','=',active_id)]</field>
-            <field name="view_id" ref="account.invoice_tree" />
+            <field name="view_id" eval="False"/>
```
Comment (dreispt) [inline contract_show_invoice/views/contract_view.xml]: You can simply remove the line.

File: contract_show_invoice/views/contract_view.xml [hunk 4]
```diff
@@ -6,10 +6,24 @@
...
             <field name="search_view_id" ref="account.view_account_invoice_filter" />
         </record>
 
+        <record id="action_contract_show_invoice_tree" model="ir.actions.act_window.view">
+            <field eval="1" name="sequence"/>
```
Comment (dreispt) [inline contract_show_invoice/views/contract_view.xml]: Please heave the `name` attribute first.

File: contract_show_invoice/views/contract_view.xml [hunk 5]
```diff
@@ -6,10 +6,24 @@
...
+            <field name="view_mode">tree</field>
+            <field name="view_id" ref="account.invoice_tree"/>
+            <field name="act_window_id" ref="act_analytic_invoices"/>
+        </record>
+
+        <record id="action_contract_show_invoice_form" model="ir.actions.act_window.view">
+            <field eval="2" name="sequence"/>
```
Comment (dreispt) [inline contract_show_invoice/views/contract_view.xml]: idem

**Comment by pedrobaeza** [comment]
Hi, Daniel, I have checked your CLA.

:+1: for the PR.

Please attend the rest of the comments (although they are not strictly needed), and I'll merge. You can make the same PR for version 9, because there's the same problem AFAIK.

**Comment by drl9319** [comment]
Thanks for your atention.

**Comment by dreispt** [comment]
LGTM :+1:

**Pull request merged**